src/pybind/mgr/volumes/module.py

```diff
@@ -210,6 +216,14 @@ class Module(orchestrator.OrchestratorClientMixin, MgrModule):
                     "and optionally, in a specific subvolume group",
             'perm': 'r'
         },
+        {
+            'cmd': 'fs subvolume exist '
+                   'name=vol_name,type=CephString '
+                   'name=group_name,type=CephString,req=false ',
+            'desc': "Check whether any subvolume is present or not in a volume "
+                    "and optionally, in a specific subvolume group",
```

Comment:
Suggest "Check a volume for the existence of a subvolume, optionally in a specified subvolume group"

Reply:
Updated!!